docs/csharp/misc/cs0076.md

```diff
@@ -2,14 +2,24 @@
 description: "Compiler Error CS0076"
 title: "Compiler Error CS0076"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "CS0076"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0076"
 ms.assetid: 8b3e8674-8bfb-4be5-a466-cdf6134e3935
 ---
 # Compiler Error CS0076
 
-The enumerator name 'value__' is reserved and cannot be used  
-  
- An item in an [enumeration](../language-reference/builtin-types/enum.md) cannot have an identifier called **value__**.
+The enumerator name 'value__' is reserved and cannot be used
+
+ An item in an [enumeration](../language-reference/builtin-types/enum.md) cannot have an identifier called **value__** since it is reserved for use as the enumeration's backing field.
```

Comment:
```suggestion
An item in an [enumeration](../language-reference/builtin-types/enum.md) cannot have an identifier called **value__** since it's reserved for use as the enumeration's backing field.
```